using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,118 @@
+multi-instance elements to iteratte over the list.
+
+![img.png](img/document-urls-collection-example.png)
+
+## Available actions
+
+The portaal taak plugin supports the following actions that can be configured in process links in order to create and 
+complete user tasks through external systems like the NL Portal.
+
+A general description on how to create process links, can be found [here](../create-process-link.md).
+
+### Create Portaal taak
+        
+The **Create portal task** action is linked to a user task that can be completed in an external system. A taak object
+is created when the task is reached. This taak object is created in the Objecten API configured for the object 
+management configuration for the plugin configuration.
+
+When creating a process link the following properties have to be entered:
+- **Form type.** The type of form to be used for completing the user task. Options are 'Form definition' to use a form 
+definition that the receiving system is supposed to understand, or 'URL' when using a url to the form definition.
+- **Formulier ID.** The identifier of the form in the receiving system. Required when using form type 'Form definition'.
+- **Formulier URL.** The URL of the form definition. Required when using form type 'URL'.
+- **Task data for the recipient.** The data in Valtimo that is relevant to the task and should be included for 
+prefilling in the task form. This fills a json object of data in the taak object. Every key is a json pointer to the
+property in the taak object data. The value is a value resolver string that is used to fill the property.
+- **Information entered by the recipient.** This is the mapping used to transfer the data received in the taak back to 
+the case. Every key is a value resolver string that is used to update the data in Valtimo. The corresponding value is a
+json pointer to the property in the taak object that contains the data.
+- **Receiver.** Indicates how to find the recipient of the task. Options are: 'Case initiator' to use the user linked by
+the zaak rol as the initiator of the case. 'Other' to manually select a recipient.
+- **Identification key.** Indicates what kind of recipient the task has. Required when choosing receiver type 'Other'.
+For example 'bsn' can be used to identify a user where BSN is the key.
+- **Identification value.** Indicates which user is the recipient of the task. Required when choosing receiver type
+'Other'. For example when Identification key 'bsn' is used, the value could be '059861095' to indicate that is the BSN 
+of the user.
+
+An example process link configuration:
+![Create portal task process link](img/configure-create-portaal-taak.png)
+
+### Complete Portaal taak
+        
+The **Complete Portaaltaak** action should be used in the `Uploaded documents handler process` to do the last step in 
+handling the update from the objects API. When executed it will complete the user task and update the status of the 
+taak object to `verwerkt`
```

Comment:
Missing a dot at the end here. Is this the end of the documentation, or is there still some uncommited code? If there's nothing to configure, it might be worth mentioning there are no additional properties that have to be configured.

Reply:
Fixed